testing/tests/DevExpress.ui.widgets.dataGrid/columnsResizingReorderingModule.tests.js

```diff
@@ -3,30 +3,54 @@ import { addShadowDomStyles } from 'core/utils/shadow_dom.js';
 QUnit.testStart(function() {
 
     const markup =
-'<style>\
+'<style nonce="qunit-test">\
     body {\
         padding: 0;\
         margin: 0;\
     }\
 </style>\
-<div style="padding: 0px 40px; margin: 0px 50px">\
+<div id="testWrapper">\
     <div id="testContainer"></div>\
 </div>\
 <div id="root">\
     <div id="container" class="dx-datagrid"></div>\
     <div id="container2" class="dx-datagrid"></div>\
 </div>\
-<div id="itemsContainer" style="font-size: 0"><div style="width:125px; display: inline-block;"></div><div style="width:125px; display: inline-block;"></div></div>\
-<div id="itemsContainerVertical"><div style="width:125px; height: 50px;" ></div><div style="width:125px; height: 50px;" ></div></div>\
+<div id="itemsContainer"><div></div><div></div></div>\
+<div id="itemsContainerVertical"><div></div><div></div></div>\
 \
 <div class="dx-swatch-1">\
     <div id="gridInSwatch">\
-    <div id="swatchitemsContainer" style="font-size: 0"><div style="width:125px; display: inline-block;"></div><div style="width:125px; display: inline-block;"></div></div>\
+    <div id="swatchitemsContainer"><div></div><div></div></div>\
```

Comment:
Let's make more obvious formatting and add classes to inner divs:

```suggestion
    <div id="swatchitemsContainer">
      <div class="swatchitemsContainer__child"></div>\
      <div class="swatchitemsContainer__child"></div>\
    </div>\
```